  File "D:\GitHub\stzlib\libraries\stzlib\test\temp\temp_x3j1r8p6.py", line 43
    import random
IndentationError: unexpected indent
